### ISSUE TITLE
Conda Verify marks valid spec containing '|' as invalid

### ISSUE THREAD

**Issue by nehaljwani** [conda/conda-verify#33]
```
C1114 Found invalid dependency "pyqt 4.11.*| 5.6.*" in info/index.json
```

**Comment by mandeep** [comment]
Thanks, I think this is related to #25

**Comment by mandeep** [comment]
@nehaljwani should the spec ``pyqt 4.11.*| 5.6.*`` be ``pyqt 4.11*| 5.6*`` instead?

https://conda.io/docs/user-guide/tasks/build-packages/package-spec.html#build-version-spec

**Comment by mandeep** [comment]
Nevermind, I've pushed a fix in #29
